### DISCUSSION TITLE
Тупит интернет после неправильной настройки zapret

### DISCUSSION THREAD

**Discussion by vovan2055** [bol-van/zapret#663] [Q&A] [answered]
Недавно пробовал настроить zapret, в процессе видимо выставил неправильные параметры и работа с интернетом в системе ухудшилась. Сейчас zapret настроен и работает, но проблема остаётся даже при остановке службы. Сайты грузятся с задержкой, команда ping в терминале также ждёт несколько секунд. Под windows на той же машине таких проблем нет. Может кто-то подсказать, что я мог в процессе сломать и как всё вернуть в норму?

**Comment by bol-van** [comment]
На пинги zapret никак не может влиять. Там icmp вообще не затрагивается.

**Reply by bol-van** [reply]
Я бы начал с проверки перегрузки системы. Не жрет ли что-то процессор под завязку.
И проверить dns. Не тупит ли он

**Reply by vovan2055** [reply]
С нагрузкой всё нормально, процессор не забит. С dns не уверен, что именно мне нужно смотреть

**Reply by bol-van** [reply]
ping lenta.ru
ping sobaka.ru
ping nnmclub.to

Не думает ли он подолгу прежде чем начинаются собственно пинги

**Reply by vovan2055** [reply]
Думает, со всеми тремя

**Reply by remittor** [reply]
Может в сторону DNS глянуть?

**Reply by vovan2055** [reply]
что конкретно смотреть?

**Reply by vovan2055** [reply]
проблема скорее всего не в dns, даже после резолва ping всё равно может ждать

**Reply by bol-van** [reply]
Роутер тоже плохо пингается ?

**Reply by vovan2055** [reply]
нет, с айпи роутера всё работает без задержек

**Reply by bol-van** [reply]
а mtr докуда нибудь что говорит ?

**Reply by vovan2055** [reply]
mtr ya.ru даёт вот это
![image](https://github.com/user-attachments/assets/52e7bea2-9a43-49b0-81b0-fba4af01a5ca)

**Reply by bol-van** [reply]
mtr шлет те же самые icmp, что и ping.
пинг реально тормозит при таком mtr ?

**Reply by vovan2055** [reply]
Только что перепровеил с тем же адресом, тормозит

**Reply by bol-van** [reply]
Не DNS, а именно пинги большие ?
Остается тогда попингать 1.1.1.1
и посмотреть за этим процессом в tcpdump -ni any host 1.1.1.1

**Reply by vovan2055** [reply]
Но и сам mtr тоже сначала ждёт, если адрес новый

**Reply by vovan2055** [reply]
```
listening on any, link-type LINUX_SLL2 (Linux cooked v2), snapshot length 262144 bytes
22:24:17.985523 wlp8s0 Out IP 192.168.1.106 > 1.1.1.1: ICMP echo request, id 10, seq 1, length 64
22:24:18.024550 wlp8s0 In  IP 1.1.1.1 > 192.168.1.106: ICMP echo reply, id 10, seq 1, length 64
22:24:18.986753 wlp8s0 Out IP 192.168.1.106 > 1.1.1.1: ICMP echo request, id 10, seq 2, length 64
22:24:19.025844 wlp8s0 In  IP 1.1.1.1 > 192.168.1.106: ICMP echo reply, id 10, seq 2, length 64
22:24:19.988008 wlp8s0 Out IP 192.168.1.106 > 1.1.1.1: ICMP echo request, id 10, seq 3, length 64
22:24:20.026326 wlp8s0 In  IP 1.1.1.1 > 192.168.1.106: ICMP echo reply, id 10, seq 3, length 64
22:24:20.989237 wlp8s0 Out IP 192.168.1.106 > 1.1.1.1: ICMP echo request, id 10, seq 4, length 64
22:24:21.028635 wlp8s0 In  IP 1.1.1.1 > 192.168.1.106: ICMP echo reply, id 10, seq 4, length 64
22:24:21.990806 wlp8s0 Out IP 192.168.1.106 > 1.1.1.1: ICMP echo request, id 10, seq 5, length 64
22:24:22.029520 wlp8s0 In  IP 1.1.1.1 > 192.168.1.106: ICMP echo reply, id 10, seq 5, length 64
22:24:22.992547 wlp8s0 Out IP 192.168.1.106 > 1.1.1.1: ICMP echo request, id 10, seq 6, length 64
22:24:23.032285 wlp8s0 In  IP 1.1.1.1 > 192.168.1.106: ICMP echo reply, id 10, seq 6, length 64
22:24:23.994420 wlp8s0 Out IP 192.168.1.106 > 1.1.1.1: ICMP echo request, id 10, seq 7, length 64
22:24:24.033337 wlp8s0 In  IP 1.1.1.1 > 192.168.1.106: ICMP echo reply, id 10, seq 7, length 64
22:24:24.995550 wlp8s0 Out IP 192.168.1.106 > 1.1.1.1: ICMP echo request, id 10, seq 8, length 64
22:24:25.036001 wlp8s0 In  IP 1.1.1.1 > 192.168.1.106: ICMP echo reply, id 10, seq 8, length 64
```

**Reply by bol-van** [reply]
Не вижу здесь никаких тормозов. Четко ответы в течение ~40 ms.
Какая вообще картинка с пингом ? Что пишется ?

**Reply by vovan2055** [reply]
С пингом 1.1.1.1 тормозов и правда нет
```
PING 1.1.1.1 (1.1.1.1) 56(84) bytes of data.
64 bytes from 1.1.1.1: icmp_seq=1 ttl=58 time=52.0 ms
64 bytes from 1.1.1.1: icmp_seq=2 ttl=58 time=47.1 ms
64 bytes from 1.1.1.1: icmp_seq=3 ttl=58 time=66.3 ms
64 bytes from 1.1.1.1: icmp_seq=4 ttl=58 time=41.6 ms
64 bytes from 1.1.1.1: icmp_seq=5 ttl=58 time=42.1 ms
64 bytes from 1.1.1.1: icmp_seq=6 ttl=58 time=46.9 ms
```
Также сейчас попробовал скачать модельку с civitai, скорость загрузки хорошая, но перед загрузкой задержка ощутимая
![image](https://github.com/user-attachments/assets/d5bfbd51-141f-4bd3-84bf-eef8a0953673)

**Reply by vovan2055** [reply]
Проблема на стороне системы, это точно. С другой системы и с другого устройства в той же сети всё работает как и должно

**Reply by bol-van** [reply]
Все же это похоже на dns
nslookup sobaka.ru 192.168.0.1
nslookup lenta.ru 192.168.0.1
тупит ?

**Reply by vovan2055** [reply]
вот тут уже интереснее
```
 ~ $ nslookup sobaka.ru 192.168.0.1
Server:		192.168.0.1
Address:	192.168.0.1#53

Non-authoritative answer:
Name:	sobaka.ru
Address: 217.65.3.230

 ~ $ nslookup lenta.ru 192.168.0.1
;; communications error to 192.168.0.1#53: timed out
Server:		192.168.0.1
Address:	192.168.0.1#53

Non-authoritative answer:
Name:	lenta.ru
Address: 81.19.72.34
Name:	lenta.ru
Address: 81.19.72.32
Name:	lenta.ru
Address: 81.19.72.33
```

**Reply by NewUse** [reply]
если в настройках браузера принудительно включить DoH тормозить перестанет?

**Reply by vovan2055** [reply]
нет, не помогло

**Reply by bol-van** [reply]
Сам DOH тоже надо заресолвить вначале.
Возможно, на роутере прописано 2 DNS, один из которых не пашет
Надо с этим разбираться

**Reply by brnsky** [reply]
> Сам DOH тоже надо заресолвить вначале.
> Возможно, на роутере прописано 2 DNS, один из которых не пашет
> Надо с этим разбираться

А если проблема в кастомном днс (DoH), то есть его вырубаю, тогда Ютуб грузиться начинает. Причем проблема только с ютубом наблюдается. Пинги до адресов идут хорошо, без задержек. Это только проблема на стороне днс получается или можно /etc подчистить?

**Comment by vovan2055** [comment]
Если это поможет, в списке сетей у меня вот такое
![image](https://github.com/user-attachments/assets/522e91f1-4bf8-4f40-b980-11683ee09c03)

**Reply by eldash666** [reply]
> Если это поможет, в списке сетей у меня вот такое ![image](https://private-user-images.githubusercontent.com/102741824/381265941-522e91f1-4bf8-4f40-b980-11683ee09c03.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzgyNTU2ODksIm5iZiI6MTczODI1NTM4OSwicGF0aCI6Ii8xMDI3NDE4MjQvMzgxMjY1OTQxLTUyMmU5MWYxLTRiZjgtNGY0MC1iOTgwLTExNjgzZWUwOWMwMy5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjUwMTMwJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI1MDEzMFQxNjQzMDlaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1lZDBjMWI5ZTI0NjM3MzMzMTQ2NjI5Njc2NThiMGQ0ZDgxMjg5NDg1ZDA1ZTc5Nzk5MzE5OGY0MTJlMWFmZDEwJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.QvQoBDWLofshQ7JIC63HP7VUenXsEmenqX13ihhHE4o)


Привет, я уже день пытаюсь инет вернуть. И винду снес, переустановил. И на роутере сброс настроек через кнопку сделал. Все равно не работает, на ноуте инет, моросит, замедленный очень. А на компе через кабель, работает инет норм

**Comment by remittor** [comment]
Про настройку DNS: https://github.com/remittor/zapret-openwrt/wiki/Tune%E2%80%90up-DNS%E2%80%90over%E2%80%90HTTPS

**Comment by vovan2055** [comment, marked as answer]
В общем, помогло подчистить конфиги в /etc. Уж не знаю, что я там мог сломать, но теперь, судя по всему, проблема пропала

**Reply by brnsky** [reply]
А какой именно конфиг чистили?

**Comment by Aboba52-hah** [comment]
Ребят, столкнулся с подобной проблемой, опыта в компе не ояень много, помогите! Скачал запрет с гид хаба гажал start выбрал запрет для ютуба и вроде все окей но когда решил зайти в роблокс в плейс инет резко пропадает и не дает подключиться, только через перезагрузку.

**Reply by brnsky** [reply]
Добавь в исключения адреса роблокс плейс

**Reply by eldash666** [reply]
> Добавь в исключения адреса роблокс плейс

Как это снести.я Винду снёс, переустановил, роутрер сбросил через кнопку, к заводским настройкам. Все равно инет замедляется на ноуте через вайфай, а на пк через кабель норм

**Reply by NewUse** [reply]
вообще никак не связанные вези, подключи ноут проводом.

**Comment by Abdulvus** [comment]
После установки новой версии я поставил zapret в автозагрузку и после запуска пк у меня интернет вообще отпал только после выключения zapret в диспетчере задач интернет снова заработал
